src/lib.rs

```diff
@@ -212,7 +210,8 @@ impl SystemRuntime {
         trace!("configure network");
         // Show available network interfaces
         //show_dir("/sys/class/net/", false);
-        let (connection, handle, _) = new_connection().unwrap();
+        let (connection, handle, _) = new_connection()
+            .expect("expected to be able to start a new rtnetlink connection");
```

Comment:
`expect` should only be used in situations where you would never actually expect an Err, or when you want "let's crash because we can't allow or handle this situation". Keep in mind that it will only crash the thread (which may not crash the program; I don't know how this function will be called). That being said, one of the two cases might be exactly what you were going for, I have no idea, in which case this is fine.

Reply:
Added a `match` for appropriate result handling

* Ok: start initialization of network via netlink
* Err:
    * log the error 
    * todo(not this PR): try to recover from the given error 
    * todo(not this PR): if auraed can not recover, abort. We should discuss how we handle unrecoverable errors during initialization. But, I also suggest to postpone the discussion for a 2nd stabilization iteration.